src/ocl/tensorocl.cpp

```diff
@@ -210,18 +225,18 @@ void OpTensor3d(const Handle& handle,
                        long(Aoffset),
                        long(Boffset),
                        long(Coffset),
-                       int(clens[1]));
+                       long(total_work),
+                       long(total_work2),
+                       int(!float_equal(miopen_beta, 0.0)),
```

Comment:
could we have used short in place of int to pass bool?

Reply:
Yes we can but it may not have much impact on performance, while int is easier to operate.

Reply:
Ok, let's keep it int.